lib/wasix/src/lib.rs

```diff
@@ -30,6 +30,9 @@ compile_error!(
     "The `js` feature must be enabled only for the `wasm32` target (either `wasm32-unknown-unknown` or `wasm32-wasi`)."
 );
 
+#[cfg(all(test, target_arch = "wasm32"))]
+wasm_bindgen_test::wasm_bindgen_test_configure!(run_in_browser);
```

Comment:
Do you think it makes sense to isolate all the web/js specific code in an `arch/web/`?

I think that would be better for code structure instead of sprinkling web related code around the crate.